### ISSUE TITLE
how to derive masks

### ISSUE THREAD

**Issue by Miner-AI** [naver-ai/BlendNeRF#1]
hey！
          Thanks for your great job! 
           While reading your paper, I have a question about your work, how do you derive the mask m and m` which are used in Step4(local alignment).
        Thanks a lot!

**Comment by blandocs** [comment]
Hi, thank you for your interest :)

Users can select editing masks or just use a pretrained segmentation network such as [BiSeNet](https://github.com/zllrunning/face-parsing.PyTorch).
